src/domain/events/index.ts

```diff
@@ -86,15 +90,16 @@ async function _addContract(
   }
 }
 
-export async function _registerNewOrder(
+async function registerNewOrder(
   event: ConditionalOrderCreatedEvent | MerkleRootSetEvent,
   composableCow: ComposableCoWInterface,
```

Comment:
Why do yo need to receive the `composableCow` if you already have it in scope?

Reply:
I didn't change this code, just removed it from the exports and removed `_` suffix

Reply:
Yes, but you created a global const with the `composableCow` instance, so no need to pass it as a parameter any more